LawGuardPro.API/Controllers/UsersController.cs

```diff
@@ -1,45 +1,54 @@
-﻿
-using LawGuardPro.Application.DTO;
+﻿using LawGuardPro.Application.DTO;
 using LawGuardPro.Application.Features.Identity.Interfaces;
 using LawGuardPro.Application.Features.Identity.Commands;
 using MediatR;
 using Microsoft.AspNetCore.Mvc;
 using System.Net;
+using Microsoft.AspNetCore.JsonPatch;
+using LawGuardPro.Application.Features.Settings.Profiles;
 
-namespace LawGuardPro.API.Controllers
+namespace LawGuardPro.API.Controllers;
+
+[Route("api/UsersAuth")]
```

Comment:
use lower-case letters like `api/usersauth`

Reply:
Updated route to use lowercase letters (api/usersauth)